config/routes.rb

```diff
@@ -1,5 +1,8 @@
 Rails.application.routes.draw do
   resources :users, only: %i[index show] do
-    resources :posts, only: %i[index show]
+    resources :posts, only: %i[index show create] do
+      resources :comments, only: %i[create]
```

Comment:
[REQUIRED] Kindly include new so that you can have a new endpoint

Reply:
Well resolved ✔️